packages/backend/src/modules/tvl/repositories/deleteArchivedRecords.ts

```diff
@@ -7,17 +7,20 @@ import { Knex } from 'knex'
 export function deleteHourlyUntil(
   knex: Knex,
   tableName: string,
-  timestamp: UnixTime,
+  to: UnixTime,
+  from: UnixTime | undefined,
 ) {
-  return (
-    knex(tableName)
-      .where('unix_timestamp', '<', timestamp.toDate())
-      // do not delete daily
-      .andWhereRaw(`extract(hour from "unix_timestamp") != 0`)
```

Comment:
Is it ok that this condition was dropped?

Reply:
mean, the `!=` one

Reply:
0 % 6 = 0 - so the first condition is included in the second condition